_docs/operations/updates.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: guide
+title: StorageOS Docs - Update StorageOS version
+anchor: operations
+module: operations/updates
+---
+
+# Update StorageOS version
```

Comment:
Can we settle on the verb 'upgrade' rather than 'update'? The former has stronger connotations of changing version I think (c.f. apt upgrade vs apt update).

Reply:
done